### DISCUSSION TITLE
Nextcloud LXC helper script produces error

### DISCUSSION THREAD

**Discussion by Sagzster** [tteck/Proxmox#2404] [Help Needed] [answered]
Tried using your helper script with the following options:
privileged container
4VCPU
8192 RAM
2048 Swap
200GB storage
 ✓ Updated Container OS
 ✓ Installed Dependencies
 - Installing NextCloudPi (Patience)  /
[ERROR] in line 23: exit code 0: while executing command "$@" > /dev/null 2>&1
The silent function has suppressed the error, run the script with verbose mode enabled, which will provide more detailed output.

Thanks,
Brad

**Comment by tteck** [comment]
What Linux distribution?

**Reply by Sagzster** [reply]
Debian 12

**Reply by tteck** [reply, marked as answer]
> - If the default Linux distribution is not adhered to, script support will be discontinued.

**Reply by Sagzster** [reply]
Copy that... I'll redo with the default... might have zoomed passed that.
